tests/test_e2e.py

```diff
@@ -405,7 +404,7 @@ def test_profile_model(
     file_storage.upload("code.py", code)
 
     model_id = "test_profile_model"
-    amr = json.load(open(f"{context_dir}/amr.json"))
+    amr = json.load(open(f"./tests/amr.json"))
```

Comment:
I think a real `AMR` could also optionally be provided here

Reply:
Ah yes, you are right! The AMR is irrelevant here it's just something we patch to. I'd forgotten it's not _really_ used by MIT